Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -1628,6 +2160,17 @@ public static void OpenALootBoxForAssetInstances(int assetInstanceID, Action<Loo
             LootLockerAPIManager.CreatingAnAssetCandidate(data, onComplete);
         }
 
+        /// <summary>
+        /// Update an existing asset candidate.
+        /// </summary>
+        /// <param name="assetId">ID of the asset candidate to update</param>
+        /// <param name="isCompleted">If true, the asset candidate will become an asset and can not be edited any more.</param>
+        /// <param name="onComplete">onComplete Action</param>
+        /// <param name="name">New name of the asset candidate</param>
+        /// <param name="kv_storage">A Dictionary<string, string> of key-values to use</param>
+        /// <param name="filters">A Dictionary<string, string> of key-values that can be used to filter out assets</param>
+        /// <param name="data_entities">A Dictionary<string, string> of data to include in the asset candidate</param>
+        /// <param name="context_id">An ID of the context to use when promoting to an asset, will be automatically filled if not provided</param>
```

Comment:
Should probably be noted that these are optional and have default values

Reply:
Yes 👍